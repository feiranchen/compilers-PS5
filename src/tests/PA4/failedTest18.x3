#Test: stage2test7
#printing before reading input
#[Stage 2]

#Program:
fun getinput() : Iterable<String>
{
	return ["Print this one before reading, too."]++input;
}

return ["Print this line before reading the input."]++getinput();

