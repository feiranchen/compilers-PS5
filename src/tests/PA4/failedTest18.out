Possible outputs: 
Print this line before reading the input.
Print this one before reading, too.
A few parts of the semantics are less than obvious.heyyy
The following constraints,heyyy
along with the above directive to implement the obvious semantics whenever such are available,heyyy
should give you enough information to complete this assignment.heyyy
Of course,heyyy
we reserve the right to add to this list when we discover that we have (inevitably) left something out.heyyy
OR
Comment: 1
Print this line before reading the input.Print this one before reading, too.A few parts of the semantics are less than obvious.heyyyThe following constraints,heyyyalong with the above directive to implement the obvious semantics whenever such are available,heyyyshould give you enough information to complete this assignment.heyyyOf course,heyyywe reserve the right to add to this list when we discover that we have (inevitably) left something out.heyyy
OR
Comment: 1
Print this line before reading the input.
Print this one before reading, too.
A few parts of the semantics are less than obvious.heyyy
